### PR TITLE
Update ops-obs doc with Pathfinder documentation

### PR DESCRIPTION
**Pull request by estevesjh**
Here it's a draft of the Pathfinder documentation page.

See ticket https://jira.lsstcorp.org/browse/DM-33431

### CODE REVIEW

**Comment by b1quint** [comment]
Can you make a new commit removing all the `.DS_Store` files, please? These are MacOs files that should not live in the repository.

**Comment by b1quint** [comment]
Can you also merge `main` into this branch to catch up on all the updates so the two are in sync?

**Comment by tribeiro** [comment]
Don't merge `main` into your branch.. rebase it instead

File: Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/_static/README [hunk 1]
```diff
@@ -0,0 +1,23 @@
+This README includes information on using static objects within this directory.
```
Comment (tribeiro) [inline Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/_static/README:1]: You may want to remove this README file here

File: Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/refrigeration-system-pathfinder.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+In this page you find information about the design, reviews and procedures of the refrigeration system, pathfinder.
+
+
+.. _Refigeration-System-Pathfinder-Procedures:
+
+Procedures
+=============
+
+Here you find some of the start up procedures of pathfinder. The information is constantly being updated. The safest way of find the procedures is to search on the docushare.
```
Comment (tribeiro) [inline Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/refrigeration-system-pathfinder.rst:24]: Please, make 1 phrase per line, this makes it easier for us to review future changes.

**Review by tribeiro — COMMENTED** [review]
Please, make sure to rebase your branch to main. 

Also, make sure you break each phrase into one line of document, as this will make it easier for us to review future changes. I left one inline comment about this but I noticed it in other places as well.

File: README [hunk 3]
```diff
@@ -3,3 +3,13 @@ This documentation is built with Sphinx (https://www.sphinx-doc.org/en/master/)
 
 See https://obs-ops.lsst.io/project/contributing.html for details on contributing to the documents.
 
+
+pipetask run -j 32 -d "detector IN (0) AND instrument='LATISS' AND \
```
Comment (tribeiro) [inline README:7]: I think this was copied here by mistake.

**Comment by estevesjh** [comment]
> Can you also merge `main` into this branch to catch up on all the updates so the two are in sync?

**Review by b1quint — CHANGES_REQUESTED** [review]
Tiago left a message asking to rephrase the documentation to have one sentence per line. This makes it easier to trace the changes. Would you do that, please? Once you do, mark the conversations as resolved.

File: Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/refrigeration-system-pathfinder.rst (hunk 2 above)
Comment (estevesjh) [reply Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/refrigeration-system-pathfinder.rst]: Done

File: README (hunk 3 above)
Comment (estevesjh) [reply README]: Done

File: Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/_static/README (hunk 1 above)
Comment (estevesjh) [reply Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/_static/README]: Done

**Comment by estevesjh** [comment]
The suggestions were implemented the readme has one sentence per line and the DS_store files were removed. 

Then to make things clearer I did a squash on unnecessary commits and I updated the description of the commit.

**Comment by tribeiro** [comment]
So, the one sentence per line actually needs to be implemented on all rst docs and not necessary on the readme files.

**Comment by estevesjh** [comment]
> So, the one sentence per line actually needs to be implemented on all rst docs and not necessary on the readme files.

I implemented on all lines, I just checked it. The description above was not precise.

**Comment by b1quint** [comment]
@estevesjh and I went through cleaning up the existing commits and I believe we are good to merge this PR now.

**Review by b1quint — APPROVED** [review]

**Pull request merged**